sphinx_gallery/gen_gallery.py

```diff
@@ -39,7 +39,7 @@
 
 DEFAULT_GALLERY_CONF = {
     'filename_pattern': re.escape(os.sep) + 'plot',
-    'ignore_pattern': '__init__\.py',
+    'ignore_pattern': r'__init__\.py',
```

Comment:
hmmm - does this mean we should add something to the docs like "if you're getting warnings you should put `r"` in front of your ignore pattern?

Reply:
This is just a general Python thing that `\` when you want it to be `\` and not a special char should either be `\\` in strings or you should use `r''` strings